aws_datalake/modules/iam/main.tf

```diff
@@ -7,6 +7,9 @@ resource "aws_iam_role" "segment_data_lake_iam_role" {
   tags               = local.tags
 }
 
+# https://www.terraform.io/docs/providers/aws/d/region.html
+data "aws_region" "current" {}
```

Comment:
This is the region where the DL is being set up right? Customers can use Segment in the US but their DL can be configured in EU. We need this to be a different config than the region the customer is setting up their DL in. Something like `segment_region` or on those lines.